see https://github.com/Cantera/cantera/issues/433 and https://github.com/Cantera/cantera/commit/11a0727d5ca6172fcb3596dbc11e7f7b1cc2e806

From 11a0727d5ca6172fcb3596dbc11e7f7b1cc2e806 Mon Sep 17 00:00:00 2001
From: Ray Speth <speth@mit.edu>
Date: Tue, 21 Feb 2017 20:29:37 -0500
Subject: [PATCH] [Test] Fix reproducibility of values used in
 add_species_sequential test

Exact floating point equality can be assured only in the case where the species
are added in the same order, since this affects summations involved in
calculating the mixture molecular weight. This resulted in test failures with
certain versions of the Intel compiler.

Resolves #433.
---
 test/data/kineticsfromscratch.cti | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/data/kineticsfromscratch.cti b/test/data/kineticsfromscratch.cti
index eaba17b..25983cf 100644
--- a/test/data/kineticsfromscratch.cti
+++ b/test/data/kineticsfromscratch.cti
@@ -2,7 +2,7 @@ units(length='m', time='s', quantity='kmol', act_energy='cal/mol')
 
 ideal_gas(name = "ohmech",
       elements = " O  H  Ar ",
-      species = """ h2o2: H2  H  O  O2  OH  H2O  HO2  H2O2  AR""",
+      species = """ h2o2: AR  O  H2  H  OH  O2  H2O  H2O2  HO2""",
       reactions = "all",
       transport = "None",
       initial_state = state(temperature = 300.0,
